DSharpPlus/Clients/DiscordClient.WebSocket.cs

```diff
@@ -490,27 +490,55 @@ internal async Task SendResumeAsync()
             };
             var resumestr = JsonConvert.SerializeObject(resume_payload);
 
-            await this.WsSendAsync(resumestr).ConfigureAwait(false);
+            await this.SendRawPayloadAsync(resumestr).ConfigureAwait(false);
         }
+
         internal async Task InternalUpdateGatewayAsync()
         {
             var info = await this.GetGatewayInfoAsync().ConfigureAwait(false);
             this.GatewayInfo = info;
             this.GatewayUri = new Uri(info.Url);
         }
 
-        internal async Task WsSendAsync(string payload)
+        internal async Task SendRawPayloadAsync(string rawJson)
         {
-            this.Logger.LogTrace(LoggerEvents.GatewayWsTx, payload);
```

Comment:
this naming was completely fine (and arguably better, payload describes what you *want* to do)